api/python/api/fit.py

```
@@ -47,27 +46,7 @@ def avg_speed_validator(cls, value: float, info: ValidationInfo) -> float:
 
 
 class LapCreate(LapBase):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)